description/src/description.rs

```diff
@@ -37,19 +49,15 @@ fn ty_description(
     transformer: &Transformer<String>,
 ) -> anyhow::Result<String> {
     let ident = ty.path.ident().unwrap_or_default();
-    let prefix = type_def_prefix(&ty.type_def);
+    let prefix = if let TypeDef::Variant(_) = ty.type_def {
+        "enum "
+    } else {
+        ""
```

Comment:
One thought I had from the prev PR; I think that if we prefix enums with `enum`, that we should also prefix structs with `struct` :)